docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
     - Decision tree point
-    - 
+    - Age, scenario 
```

Comment:
Does ANC attendance change with age?

Reply:
From the V&V notebook, it looks like the ANC values we have in the artifact don't vary by age, so I'm not sure whether we need it here.

But if we do say to stratify by age, does that mean we want the age group of the pregnant simulant? I think we should be as explicit as possible.

Reply:
Yeah I added a row for age of pregnant simulant at the end of the table, but maybe I should change it to GBD age group, as you said in your other comment!